dist/gaia-header.js

```diff
@@ -648,8 +683,7 @@ module.exports = component.register('gaia-header', {
     // Elements
     this.els = {
       actionButton: this.shadowRoot.querySelector('.action-button'),
-      buttons: this.querySelectorAll('button, a'),
-      titles: this.querySelectorAll('h1')
+      titles: this.getElementsByTagName('h1')
     };
 
     // Events
```

Comment:
I think the other changes to the dist and es5 versions comes from changes to some dependencies (I ran npm update and bower update).